ipywidgets/widgets/widget.py

```diff
@@ -748,3 +749,22 @@ def _gen_repr_from_keys(self, keys):
             for key in keys
         )
         return '%s(%s)' % (class_name, signature)
+
+    def _fallback_html(self):
+        return _FALLBACK_HTML_TEMPLATE.format(widget_type=type(self).__name__)
+
+
+_FALLBACK_HTML_TEMPLATE = """\
+<p>Failed to display Jupyter Widget of type <code>{widget_type}</code>.</p>
+<p>
+  If you're reading this message in Jupyter Notebook or JupyterLab, it probably means that
+  the widgets Javascript library is either not installed or not enabled. See
```

Comment:
Should this be JavaScript, rather than Javascript -- that's how [Wikipedia](https://en.wikipedia.org/wiki/JavaScript) spells it.